File: 20zadPrimeNumbersinInterval.js
let number = 2;
let numberTo = 20;
let counter = 0;
for(number; number<numberTo; number++){

for(i=1;i<=number;i++){
    if(number/i == Math.trunc(number/i)){
        counter++;
    }
}
if(counter > 2 || number === 1){
    //ne e prosto
    counter = 0;
}
else{
    //prosto
    console.log(number);
    counter = 0;
}
}

File: 20zadPrimeNumbersInInterval.py
import math
number = 2
numberTo = 20
counter = 0

for number in range(number,numberTo):
    for i in range(1,number+1):
        if number/i == math.trunc(number/i):
            counter +=1
    if counter > 2 or number == 1:
        #doesn't print
        counter = 0
    else:
        counter = 0
        print(number)